Makefile

```diff
@@ -177,20 +189,15 @@ dummy_oauth_api: openapi-to-go-server
 			--api_folder /resources/output/api
 # ---
 
-.PHONY: install-staticcheck
-install-staticcheck:
-	go install honnef.co/go/tools/cmd/staticcheck
+.PHONY: check-dss
+check-dss: evaluate-tanka test-go-units test-go-units-crdb build-dss build-monitoring test-e2e
 
-.PHONY: staticcheck
-staticcheck: install-staticcheck
-	staticcheck -go 1.12 ./...
-
-.PHONY: test
-test:
+.PHONY: test-go-units
+test-go-units:
 	go test -ldflags "$(LDFLAGS)" -count=1 -v ./pkg/... ./cmds/...
 
-.PHONY: test-cockroach
-test-cockroach: cleanup-test-cockroach
+.PHONY: test-go-units-crdb
+test-go-units-crdb: cleanup-test-go-units-crdb
 	@docker run -d --name dss-crdb-for-testing -p 26257:26257 -p 8080:8080  cockroachdb/cockroach:v21.2.7 start-single-node --insecure > /dev/null
 	go run ./cmds/db-manager/main.go --schemas_dir ./build/deploy/db_schemas/rid --db_version latest --cockroach_host localhost
 	go test -count=1 -v ./pkg/rid/store/cockroach --cockroach_host localhost --cockroach_port 26257 cockroach_ssl_mode disable --cockroach_user root --cockroach_db_name rid --schemas_dir db-schemas/rid
```

Comment:
This check fails if those dashes are added, and it fails if the argument is removed.  Are you able to run the target successfully with the dashes?  What Go version do you have installed?  (`go version`)

Reply:
Ok, I think I've gotten to the bottom of this now.  The `schema_dir` flag isn't valid for the `go test` commands here because it's only defined for db-manager.  That error was being hidden by the missing dashes which I think was causing `go test` to treat the remaining flags like plain arguments and not flags.  Just adding the dashes reveals the `schema_dirs` error.  I added the dashes and resolved the error by removing the `schema_dirs` flag.  I also removed the `go test` for SCD because there are no test files there, so that line wasn't doing anything (and therefore also not detecting bad flags).